packages/server/src/api/controllers/deploy/index.js

```diff
@@ -226,7 +191,7 @@ exports.deployApp = async function(ctx) {
     status: DeploymentStatus.PENDING,
   })
 
-  await deployApp({
+  deployApp({
```

Comment:
Should we just let this go? If function is async at all we should probably handle error etc

Reply:
I did it so that we could make it remain an async process without the client sitting waiting for the whole deployment to finish. If it fails, the UI will still get updated since it's polling